python/cudf/cudf/core/series.py

```diff
@@ -5216,3 +5212,16 @@ def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
         result_col[equal_nulls] = True
 
     return Series(result_col, index=index)
+
+
+def _factorize(obj, na_sentinel=-1):
+
+    obj = cudf.Series(obj)
+
+    cats = obj.dropna().unique().astype(obj.dtype)
+
+    name = obj.name  # label_encoding mutates self.name
+    labels = obj.label_encoding(cats=cats, na_sentinel=na_sentinel)
+    obj.name = name
+
+    return labels, cats
```

Comment:
Can this be moved to `Frame` or `Column` and then used from there?

Reply:
> I think this is missing the top level `cudf.factorize` function?

Weird - I definitely updated `__init__.py` but it's not showing up here for some reason. checking into it

Reply:
It seemed weird in Frame - but I think column should work here. Updates to follow.